x-pack/filebeat/module/cyberarkpas/_meta/assets/elastic-json-v1.0.xsl

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is necessary to configure the Vault, but it's not used by the module. Added it here so that it's under version control. We need to see how to distribute it.